test/screenshotter/ss_data.yaml

```diff
@@ -95,7 +95,7 @@ LimitControls: |
     \displaystyle\int\limits_2^3 3x^2\,dx + \sum\nolimits^n_{i=1}i +
     \textstyle\int\limits_x^y z
 MathAtom: a\mathrel{\mathop{=}\limits^{\blue ?}}b
-MathAtom2: \mathop{\overline\mathrm{lim}}\limits_{x\to\infty}f(x)
+MathAtom2: \mathop{\overline{\mathrm{lim}}}\limits_{x\to\infty}f(x)
```

Comment:
I guess so. I wonder why the old version works at all in KaTeX. Suggests something is wrong there. Might be worth having a separate ticket for that.